file-browser.lua

```diff
@@ -237,12 +237,12 @@ end
 
 --returns the file extension of the given file
 local function get_extension(filename, def)
-    return filename:match("%.([^%./]+)$") or def
+    return string.lower(filename):match("%.([^%./]+)$") or def
```

Comment:
```suggestion
    return filename:lower():match("%.([^%./]+)$") or def
```

Using this syntax, it matches the syntax I've used elsewhere.

Reply:
Done.